### PR TITLE
Admin Actions Log: `pulsar-gpp-compiler`

### PR DESCRIPTION
**Pull request by confused-Techie**
### Requirements 

* Filling out the template is required.
* All new code requires tests to ensure against regressions.
  - However, if your PR contains zero code changes, feel free to select the checkmark below to indicate so.

* [ ] Have you ran tests against this code?
* [X] This PR contains zero code changes.

### Description of the Change

These notes detail the allowance of republication of a reserved package name.

As this is a novel event, the details here are rather thorough to ensure peace of mind is kept amongst users of the platform, to the possibility of security risks.

### CODE REVIEW

File: docs/reference/Admin_Actions.md [hunk 1]
```diff
@@ -4,6 +4,51 @@ When you consider that most backend services are a black box of code and decisio
 
 With that said this document will serve as the ongoing history of administrative actions that must be taken against the backend.
 
+## 2023 - September 4
+
+### pulsar-gpp-compiler
+
+The maintainers of the community package `pulsar-gpp-compiler` during the course of maintaining their package, had accidentally completely unpublished the package, rather than unpublishing a specific version.
+
+These maintainers had wanted to retain the same package name during a republication, but the backend reserves all package names after unpublishing to help keep users safe from a potentially [Supply Chain Vulnerability](https://en.wikipedia.org/wiki/Supply_chain_attack).
```
Comment (Daeraxa) [inline docs/reference/Admin_Actions.md:13]:
```suggestion
These maintainers had wanted to retain the same package name during a republication, but the backend reserves all package names after unpublishing to help keep users safe from a potential [Supply Chain Vulnerability](https://en.wikipedia.org/wiki/Supply_chain_attack).
```

File: docs/reference/Admin_Actions.md [hunk 2]
```diff
@@ -4,6 +4,51 @@ When you consider that most backend services are a black box of code and decisio
...
+
+Beyond that we had to confirm the following details:
+
+1. The user requesting this service had sufficient permission to the effected repository.
+2. No major, malicious changes had occurred in the codebase, which could signal a GitHub users account has been compromised.
+3. That absolutely 0 downloads have ever occurred of this package.
+
+To confirm the above:
+
+1. The user was asked to create a branch of the effected repo named `allow-pulsar-un-reserving`
```
Comment (Daeraxa) [inline docs/reference/Admin_Actions.md:29]:
```suggestion
1. The user was asked to create a branch of the affected repo named `allow-pulsar-un-reserving`
```

File: docs/reference/Admin_Actions.md [hunk 3]
```diff
@@ -4,6 +4,51 @@ When you consider that most backend services are a black box of code and decisio
...
+
+Since the nature of a Supply Chain Vulnerability relies on users having previously installed the potential package, and due to the young age of the package, we realized that if we can confirm with 100% certainty that absolutely 0 users had installed this package, nobody could be affected by a supply chain vulnerability, and it could be safe to the overall community to allow republication.
+
+An important note about confirming the download count: Because when a package is unpublished, the download count is permanently deleted from the Pulsar Package Repository, we have to rely on the logs of the backend. These logs only go back 30 days, meaning that if a package was published more than 30 days ago, it would **never** be eligible for allowing republication of a reserved name.
+
+Beyond that we had to confirm the following details:
+
+1. The user requesting this service had sufficient permission to the effected repository.
+2. No major, malicious changes had occurred in the codebase, which could signal a GitHub users account has been compromised.
```
Comment (Daeraxa) [inline docs/reference/Admin_Actions.md:24]:
```suggestion
2. No major, malicious changes had occurred in the codebase, which could signal a GitHub user's account had been compromised.
```

**Review by Daeraxa — COMMENTED** [review]
Couple of tiny errors. Only other comment is use of maintainers vs maintainer - not sure what is correct but not really important either way.

**Comment by confused-Techie** [comment]
@Daeraxa 
Thanks a ton for the review, and cleaning up my words here.

As for the difference between plural or singular, I think the singular is actually accurate, but not actually total positive.

File: docs/reference/Admin_Actions.md [hunk 4]
```diff
@@ -4,6 +4,51 @@ When you consider that most backend services are a black box of code and decisio
...
+
+At the request of this package's maintainers we investigated any way to allow the same name to be published, while keeping our users safe.
+
+Since the nature of a Supply Chain Vulnerability relies on users having previously installed the potential package, and due to the young age of the package, we realized that if we can confirm with 100% certainty that absolutely 0 users had installed this package, nobody could be affected by a supply chain vulnerability, and it could be safe to the overall community to allow republication.
+
+An important note about confirming the download count: Because when a package is unpublished, the download count is permanently deleted from the Pulsar Package Repository, we have to rely on the logs of the backend. These logs only go back 30 days, meaning that if a package was published more than 30 days ago, it would **never** be eligible for allowing republication of a reserved name.
+
+Beyond that we had to confirm the following details:
+
+1. The user requesting this service had sufficient permission to the effected repository.
```
Comment (Daeraxa) [inline docs/reference/Admin_Actions.md:23]:
```suggestion
1. The user requesting this service had sufficient permission to the affected repository.
```

Comment (Daeraxa) [reply docs/reference/Admin_Actions.md:23]: sorry one last one I found

Comment (confused-Techie) [reply docs/reference/Admin_Actions.md]: Thanks for taking a second look

**Review by Daeraxa — APPROVED** [review]

**Comment by confused-Techie** [comment]
With @Daeraxa's approval, I'll go ahead and merge this one

**Pull request merged**